plugins/run/internal-functions

```diff
@@ -21,6 +21,10 @@ fn-run() {
         export DOKKU_DISABLE_TTY=true
         shift
         ;;
+      --force-tty)
+        export DOKKU_FORCE_TTY=true
+        shift
+        ;;
```

Comment:
Might be worth adding some error checking after this switch statement to ensure both `DOKKU_DISABLE_TTY` and `DOKKU_FORCE_TTY` aren't set.

Reply:
In has_tty() I put the force-tty check first, since it is more explicit and not used as default anywhere, so it won't matter if both are set. Maybe check the final variables and add a warning in has_tty, so it's not run-specific only?

Reply:
I'd rather we exit abnormally immediately in places that depend on this behavior (so, this plugin) vs a warning that then has potentially unspecified behavior.